pkgs/development/interpreters/python/hooks/python-namespaces-hook.sh

```diff
@@ -30,9 +37,9 @@ pythonNamespacesHook() {
             # event of a "meta-package" package, which will just install
             # other packages, but not produce anything in site-packages
             # besides meta information
-            if [ -d "${constructedPath}/../" -a -z ${dontRemovePth-} ]; then
+            if [[ -d "${constructedPath}/../" ]] && [[ -z "${dontRemovePth-}" ]]; then
```

Comment:
Is there a reason to move to `[[ … ]]` here but not in the other hooks?

Reply:
I just forgot to move to `[[  ... ]]` when linting `pythonImportsCheckHook`.

Reply:
> I just forgot to move to `[[ ... ]]` when linting `pythonImportsCheckHook`.

Applied.